src/audio/codec_adapter/codec/cadence.c

```diff
@@ -22,6 +22,42 @@ static struct cadence_api cadence_api_table[] = {
 		.id = 0x01,
 		.api = cadence_api_function
 	},
+#ifdef CONFIG_CADENCE_CODEC_AAC_DEC
```

Comment:
@mrajwa we might also want to add an ``ifdef`` for ``cadence_api_function``, I couldn't find a proper name for it. But we can do it in a separate patch.

Reply:
@mrajwa  later edit, added  a separate commit for `cadence_api_function` in this way we avoid compilation errors.